Arrowgene.Ddon.GameServer/Quests/Quest.cs

```diff
@@ -332,6 +334,41 @@ public virtual CDataLightQuestList ToCDataLightQuestList(uint step)
             return result;
         }
 
+        public virtual CDataSetQuestInfoList ToCDataSetQuestInfoList()
+        {
+            var result = new CDataSetQuestInfoList()
+            {
+                QuestScheduleId = (uint)QuestScheduleId,
+                QuestId = (uint)QuestId,
+                ImageId = NewsImageId, // Optional, client has its own defaults if you fail to provide one.
+                BaseLevel = BaseLevel,
+                IsDiscovery = true, // If false, hides quest details from the news report.
+                EndDistributionDate = uint.MaxValue, // ulong.MaxValue causes some math on the client to overflow and report it as ending soon, so we use uint here.
```

Comment:
The value could be interpreted by the client as signed at some point, we've seen this before. Can you try with long.MaxValue instead of ulong?